content/docs/develop/reference/cli/index.md

```diff
@@ -241,18 +244,45 @@ $ keptn create project sockshop shipyard.yml
 
 After creating a project which represents a repository in your GitHub organization, the keptn&nbsp;CLI allows to onboard services into this project. Please note that for describing the Kubernetes resources, [Helm charts](https://helm.sh/) are used. Therefore, the keptn CLI allows setting a Helm values description in the before created project. Optionally, the user can also provide a Helm deployment and service description.
 
-To onboard a service, use the command `onboard service` and provide the project name, the Helm chart values and optionally also deployment and service descriptions.
+To onboard a service, use the command `onboard service` and provide the project name (flag `--project`), the Helm chart values (flag `--values`) and optionally also deployment (flag `--deployment`) and service (flag `--service`) descriptions.
 
 ```console
-$ keptn onboard service --project=sockshop --values=values.yaml
+$ keptn onboard service --project=sockshop --values=values_carts.yaml
 ```
 or
 ```console
-$ keptn onboard service --project=sockshop --values=values.yaml --deployment=deployment.yaml --service=service.yaml
+$ keptn onboard service --project=sockshop --values=values_carts_db.yaml --deployment=deployment_carts_db.yaml --service=service_carts_db.yaml
 ```
 
 To start onboarding a service, please see the [Onboarding a Service](../../usecases/onboard-carts-service) use case.
 
+## keptn send new-artifact
+
+After onboarding a service, the keptn&nbsp;CLI allows pushing a new artifact for the service.
+This artifact is a Docker image, which can be located at Docker Hub, Quay, or any other registry storing docker images.
+The new artifact is pushed in the first stage specified in the  `shipyard.yaml` file (usually this will be the dev-stage).
+Afterwards, keptn takes care of deploying this new artifact.
+
+To push a new artifact, use the command `send new-artifact`, which sends a new-artifact-event 
+to the keptn installation in order to deploy a new artifact for the specified service in the provided project.
+Therefore, this command takes the project (flag `--project`), the service (flag `--service`) 
+as well as the image (flag `--image`) and tag (flag `--tag`) of the new artifact.
+
+```console
+$ keptn new-artifact --project=sockshop --service=carts --image=docker.io/keptnexamples/carts --tag=0.7.0
+```
+
+## keptn send 
+
+This command allows sending arbitrary keptn events. These events have to follow the [Cloud Events](https://cloudevents.io/)
+specification and are written in JSON.
+**Note:** This command is not required for any use case and requires precise keptn event definitions, which we will
+describe in one of the next releases.
+
+To send an arbitrary keptn event, use the command `send` and pass the file containing the event (flag `--event`).
```

Comment:
a link to our cloudevents in keptn/keptn-specification and maybe one example that shows the content of such an event.json would be helpful